partcad/src/partcad/user_config.py

```
@@ -10,143 +10,109 @@
+        self.set_config_type("yaml")
 
         config_path = os.path.join(
             UserConfig.get_config_dir(),
             "config.yaml",
         )
         if os.path.exists(config_path):
             try:
-                self.config_obj = yaml.safe_load(open(config_path, "r"))
-                if self.config_obj is None:
-                    self.config_obj = {}
+                self.read_config(open(config_path, "r"))
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)